ElectronClient/gui/NoteEditor/NoteBody/CodeMirror/utils/useJoplinMode.ts

```diff
@@ -1,5 +1,6 @@
 import 'codemirror/addon/mode/multiplex';
 import 'codemirror/mode/stex/stex';
+const Setting = require('../../../../../lib/models/Setting.js');
```

Comment:
Almost there! 

However, you don't need to require Settings using the relative levels. For example check `ElectronClient/gui/NoteEditor/NoteBody/CodeMirror/CodeMirror.tsx` for an example of how `Setting` should be required.

Reply:
So I checked that. But the lib folder is in a different folder directory. So shouldn't the ideal way be to use `const Setting = require('../../../../../lib/models/Setting.js');` and not `const Setting = require('lib/models/Setting.js');` @CalebJohn

Reply:
Done. Corrected it @CalebJohn